zerver/tests/test_docs.py

```diff
@@ -142,7 +142,7 @@ def test_doc_endpoints(self) -> None:
         self._test("/api/get-events", "dont_block")
         self._test("/api/delete-queue", "Delete a previously registered queue")
         self._test("/api/update-message", "propagate_mode")
-        self._test("/api/get-own-user", "takes no parameters")
+        self._test("/api/get-own-user", "does not accept any parameters.")
```

Comment:
We don't want to have a commit where master fails tests, which the previous setup arranges.  We can address this by just moving this before the previous commit, and manually changing that file to use the standard string.  I'll do that while merging.